index.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+$db = new PDO('mysql:host=db; dbname=collection_app', 'root', 'password');
+$db->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
+$query = $db->prepare("SELECT `Kdrama`, `ReleaseYear`, `Synopsis`, `Starring` FROM `Kdramas`;");
+$query->execute();
+$Kdramas = $query->fetchAll();
```

Comment:
maybe could be functionalised to make it testable? Totally optional though I think, obvs

Reply:
I ended up having to do it as you'll probably end up using this code twice